### DISCUSSION TITLE
TypeError: Cannot read property 'set' of undefined or null at <eval>:3:11(8)

### DISCUSSION THREAD

**Discussion by atilladeniz** [pocketbase/pocketbase#5679] [Q&A] [answered]
data.userAgent | curl/8.7.1
-- | --
data.error | Something went wrong while processing your request.
data.details | TypeError: Cannot read property 'set' of undefined or null at <eval>:3:11(8)

When i try to register a user with curl:

   curl -X POST https://localhost:8090/api/collections/users/records \
  -H "Content-Type: application/json" \
  -d '{
    "username": "test_username",
    "email": "test@example.com",
    "password": "12345678",
    "passwordConfirm": "12345678",
    "displayName": "Test User",
    "emailVisibility": true,
    "verified": false,
    "role": "User"
  }'

**Comment by ganigeorgiev** [comment, marked as answer]
I don't think anyone will be able to assist you without at least some minimal reproducible steps.

Based on the error message, it is most likely triggered by a custom JS hooks code so double check your existing `pb_hooks` (if you have any).

If you are still not able to identify the culprit, please try to create a minimal reproducible and report with what version of PocketBase you are testing.

**Reply by atilladeniz** [reply]
Hey Gani!
My bad, sorry about that! I've managed to reproduce the issue now - it seems to be related to the hook.
Thanks so much for your time and help! By the way, I just have to say: We absolutely love Pocketbase! It's really fantastic to work with.
Best regards and thanks again!
